packages/react-core/src/components/AlertGroup/examples/AlertGroup.md

```diff
@@ -6,37 +6,73 @@ propComponents: ['Alert', 'AlertGroup', 'AlertActionCloseButton', 'AlertActionLi
+
+After a specified number of alerts displayed is reached, users will see an overflow message instead of new alerts. In the following example, an overflow message will appear when more than four alerts are queued.
+
+When the overflow message appears in an `AlertGroup` with `isLiveRegion`, the "view 1 more alert" text label will be read, but the alert message will not be read. 
+
+Screen reader and keyboard users will need a way to navigate to and reveal the hidden alerts before they disappear. Alternatively, there should be a place where notifications or alerts are collected to be viewed or read later. 
+
 ```ts file="AlertGroupToastOverflowCapture.tsx" isBeta
 ```
+### Dynamic alert groups
+
+Dynamic alerts are generated after the page initially loads and appear within the normal page content flow. Additional measures are required to ensure these alerts are accessible to screen readers.
+
+Click the buttons in the example below to add different alert variants to a group. 
 
-### Singular dynamic alert group
-This alert will appear in the page, most likely in response to a user action.
 ```ts file="./AlertGroupSingularDynamic.tsx"
 ```
 
-### Singular dynamic alert group with overflow message
-This alert will appear in the page, most likely in response to a user action. In this example we are showing a max of 4 alerts.
+### Dynamic alert group with overflow message
+
+In the following example, there can be a max of four alerts shown at once, based on the value of `maxDisplayed`. If there are more than the set maximum of alerts on a page, an overflow message will appear, which users can click on to see the full list of alerts. You may adjust the maximum number of allowed alerts as needed.
+
+
 ```ts file="AlertGroupSingularDynamicOverflow.tsx" isBeta
 ```
 
-### Multiple dynamic alert group
-These alerts will appear in the page, most likely in response to a user action.
+### Multiple dynamic alert groups
+
+You may generate multiple alerts within a group at once. Click the "add alert collection" button in the example below to add three toast alerts to a group at once.
+
 ```ts file="./AlertGroupMultipleDynamic.tsx"
 ```
 
-### Async alert group
-This shows how an alert could be triggered by an asynchronous event in the application. Note how you can customize how the alert will be announced to assistive technology. See the alert group accessibility tab for more information.
+### Asynchronous alert groups
```

Comment:
Also based on Matt's comment about this example, maybe this could be moved to be grouped along with the other toast group examples.